src/runtime/virtcontainers/hypervisor.go

```diff
@@ -70,6 +68,10 @@ const (
 	MinHypervisorMemory = 256
 )
 
+var (
+	hvLogger = logrus.WithField("source", "virtcontainers/hypervisor")
```

Comment:
This will change once we move into its own package right

Reply:
yes, correct